src/FakeItEasy/Core/DelegateRaiser.cs

```diff
@@ -5,7 +5,7 @@ namespace FakeItEasy.Core
     using System.Diagnostics.CodeAnalysis;
     using System.Linq;
     using System.Reflection;
-
+    using Castle.DynamicProxy.Generators;

```

Comment:
Ah. Not true. I think it came from the change to `nameof(Raise)` that I incompletely reverted.